app/uk/gov/hmrc/plasticpackagingtax/registration/models/registration/LiabilityDetails.scala

```diff
@@ -23,7 +23,8 @@ import uk.gov.hmrc.plasticpackagingtax.registration.views.model.TaskStatus
 case class LiabilityDetails(
   weight: Option[LiabilityWeight] = None,
   startDate: Option[Date] = None,
-  isLiable: Option[Boolean] = None
+  isLiable: Option[Boolean] = None,
+  expectToExceedThresholdSoon: Option[Boolean] = None // TODO: update with real name (PPTP-1027)
```

Comment:
Don't we need this flag as part of the isCompleted or isInProgress.. to return the status??

Reply:
Which one are you referring to? The `isLiable` was added by Paulo as part of his work on PPTP-1020 and is intended to hold the value supplied by the user on the new "Will you still be liable" page (I think). I added the other one as a placeholder until your work on PPTP-1027 was complete.

Reply:
Yeah... got it after I saw the review comments on my ticket :)